src/index.js

```diff
@@ -24,17 +27,58 @@ const fetchData = () => {
       tripsData: dataSet[1].trips,
       destinationsData: dataSet[2].destinations, 
     }).then(dataSet => {
-      travelers = new AllTravelers(dataSet.travelersData); 
-      //traveler instantiation
-      //agent instantiation 
+      allTravelers = new AllTravelers(dataSet.travelersData, dataSet.tripsData, dataSet.destinationsData); 
+      traveler = new Traveler(2, dataSet.travelersData, dataSet.tripsData, dataSet.destinationsData);
+      domUpdates.displayWelcome(traveler);
+      domUpdates.displayTravelersTrips(traveler);
+      tripDestinations.addEventListener('click', domUpdates.displayDestinationList(allTravelers));     
+      submitBtn.addEventListener('click', function() {
+        // domUpdates.displayNewTripCost(traveler);
+        postTrip(allTravelers, traveler);
+      });
+      domUpdates.displayTravelerCosts(traveler);
     })
-    .catch(error => console.log(error.message))
+    .catch(error => console.log(error.message));
 }
 
 const loginButton = document.getElementById('login-button')
-loginButton.addEventListener('click', domUpdates.submitLogin);   
+const tripDestinations= document.getElementById('trip-destinations')
+const submitBtn = document.getElementById('submit-btn');
+
+loginButton.addEventListener('click', domUpdates.submitLogin); 
+
+function postTrip(allTravelers, traveler) {
```

Comment:
This function is pretty big, maybe you could take out the logic of tripDetails and put it in another function?